blocks.md

```diff
@@ -1284,6 +1284,8 @@ Key | Values | Required | Default
 `max_vol` | Max volume in percent that can be set via scrolling. Note it can still be set above this value if changed by another application. | No | `None`
 `on_click` | Shell command to run when the sound block is clicked. | No | None
 `show_volume_when_muted` | Show the volume even if it is currently muted. | No | `false`
+`max_width` | Max width of output\_name | No | `None`
```

Comment:
This could perhaps be covered by #1112

Reply:
Yes, I have plans to implement this.

Reply:
Awesome!

Reply:
Nice

Reply:
Should I remove this then or leave it to be removed when #1112 gets merged?

Reply:
To be honest, it would be better for me if you remove it, because it will make things a little simpler. However, I'm not sure when I'll finish that PR.

Reply:
I think will leave for now. Is there anything specific one can help you on #1112? (If only merging to upstream when this will be merged before #1112 ^^)

Reply:
> Is there anything specific one can help you on #1112?

Not anything in particular yet, but, testing is appreciated.

Reply:
Implemented in https://github.com/greshake/i3status-rust/pull/1112/commits/5e3eec707bd76577abf402c26df3dd18ffe7a3dd